frontend/src/components/Dashbard.tsx

```diff
@@ -140,7 +140,7 @@ export const Dashboard = (): JSX.Element => {
 						Dataset</Button></Box>
 				<div className="inner-container">
 					<Grid container spacing={4}>
-						<Grid item xs>
+						<Grid item>
```

Comment:
I still saw the same issue
<img width="1038" alt="image" src="https://user-images.githubusercontent.com/13950475/194941432-4aa6ebb9-92b5-49c3-b1a9-3c473975d9a2.png">

I think last time you tried xs to fix it; but this time it won't work for me

Reply:
I made a new file called Explore.tsx to make it clear that that is the explore page. I fixed it there and remove Dasboard.tsx file to not create confusion.